sandbox/src/routes/retrieveAdviceAndGuidanceWorklist.js

```diff
@@ -0,0 +1,31 @@
+const mockResponseProvider = require('../services/mockResponseProvider')
+const businessFunctionValidator = require('../services/businessFunctionValidator')
+
+module.exports = [
+  /**
+   * Sandbox implementation for retrieve advice and guidance (CommunicationRequest) worklist endpoint
+   */
+  {
+    method: 'POST',
+    path: '/FHIR/STU3/CommunicationRequest/$ers.fetchworklist',
+    handler: (request, h) => {
+
+      const allowedBusinessFunctions = ["SERVICE_PROVIDER_CLINICIAN", "SERVICE_PROVIDER_CLINICIAN_ADMIN", "SERVICE_PROVIDER_ADMIN"]
```

Comment:
We need to be careful around the SPCA business function I think, currently SPCA won't be supported via the API-M platform (though this is dependent on what happens with ERSSUP-54035)

Reply:
Had a chat about it, for now we will continue adding SPCA in until we make a final decision on ERSSUP-54035